### PR TITLE
server settings as PBO

### PR DESCRIPTION
**Pull request by Fusselwurm**
cement CBA settings (that includes settings from ACE and TFAR) with a PBO ; will replace the current `/userconfig/cba_settings.sqf`

* enables us to have different settings (via different modset) for our various server instances
* be transparent about changes, use existing mod dev process for maximum participation
* reduce work for devops

**plz participate by suggesting more settings we should put in here (as long as they differ from the defaults)** @gruppe-adler/adler

### CODE REVIEW

File: addons/cba_settings_userconfig/config.cpp [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    class cba_settings_userconfig {
+        author = "$STR_CBA_Author";
+        name = "$STR_CBA_Settings_Component";
+        url = "$STR_CBA_URL";
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = 1.0;
+        requiredAddons[] = {"cba_settings"};
+        version = 1.0;
+        authors[] = {"commy2"};
```
Comment (Fusselwurm) [inline addons/cba_settings_userconfig/config.cpp]: :thinking:

**Comment by Fusselwurm** [comment]
:thinking: btw, are the keys here case sensitive? if not, we should probably straight away lowercase everything for easier sorting

File: addons/cba_settings_userconfig/config.cpp (hunk 1 above)
Comment (chris579) [reply addons/cba_settings_userconfig/config.cpp]: it's a template, see no reason to change this.

**Comment by chris579** [comment]
It's case insensitive as it uses `getVariable`/`setVariable`
https://github.com/CBATeam/CBA_A3/blob/master/addons/settings/fnc_get.sqf#L28

**Comment by chris579** [comment]
Would suggest following settings
```
force ace_zeus_zeusAscension = false;
force ace_zeus_zeusBird = false;
force ace_zeus_remoteWind = false;
force ace_zeus_radioOrdnance = false;
force ace_zeus_revealMines = 0;
force ace_zeus_autoAddObjects = true;
force ace_markers_movableMarkersEnabled = true;
force ace_markers_moveRestriction = 0;
```

**Comment by chris579** [comment]
Kann man wohl schließen oder?

**Pull request closed without merging**